core/src/test/java/kafka/test/junit/ClusterTestExtensions.java

```diff
@@ -91,10 +91,7 @@ public Stream<TestTemplateInvocationContext> provideTestTemplateInvocationContex
         // Process the @ClusterTemplate annotation
         ClusterTemplate clusterTemplateAnnot = context.getRequiredTestMethod().getDeclaredAnnotation(ClusterTemplate.class);
         if (clusterTemplateAnnot != null) {
-            processClusterTemplate(context, clusterTemplateAnnot, generatedContexts::add);
```

Comment:
We don't need those changes in this PR, right? That can be addressed in https://issues.apache.org/jira/browse/KAFKA-16654